docker-compose.yml

```diff
@@ -169,26 +176,42 @@ services:
       - "./pg/dev-db:/var/lib/postgresql/data/"
 
   psql:
-    extends: postgres
+    extends: postgres-persisted
```

Comment:
I think this should rather extend `postgres` and link `postgres-persisted` and use this link in the command. Same for `drop-db`.